### DISCUSSION TITLE
SQLite download on Mac

### DISCUSSION THREAD

**Discussion by delia19971111** [pawelsalawa/sqlitestudio#3933] [Q&A]
I tried to download the SQLite studio on Mac, but the system will prevent it and send a warning to me. Even I unlocked the prevention and finished the download process, I still cannot open this application.

**Comment by pawelsalawa** [comment]
This is due to fact that Mac binaries are not signed with certificate from Apple.
You can run such binary by right-clicking on it, picking "Open" from menu and then you will see the same warning about untrusted application. But this time you will have a button to run it anyway.

I've created #3932 to perhaps sign binaries with certificates in future, so this problem goes away.
